### ISSUE TITLE
[plugin_manager] - hmail_password in plugin_manager enabled but Account Administration link is not showing up

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#338]
```
Token: 5f52f9feadb74870edbbac918328f9e4 (Don't modify this token!)
Version: 2.5 (12-05-2012)
PHP: 5.2.11
RCMAIL: 0.7.2
Database: mysql
SERVER: Apache
----
I.  Issue Description:

I have installed the plugin and its dependency ("settings" plugin) through the 
plugin manager. I have also enabled the DCOM according to the instructions (but 
not for the Apache user, as it does not exist on my system. Instead I enabled 
DCOM for the user the Apache process run under). After selecting it in the 
Manage Plugins->Miscellaneous->Change Password, the popup message that the 
change is saved is displayed but the setting is nowhere to be found.

II. Steps to reproduce the Issue:
1.
2.
3.

```

Original issue reported on code.google.com by `miket...@gmail.com` on 18 May 2012 at 10:35

**Comment by GoogleCodeExporter** [comment]
```
Do you use hMailserver? If so, is hMailserver running on the same box as 
Roundcube?
```

Original comment by `rol...@liebl.ath.cx` on 18 May 2012 at 10:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 18 May 2012 at 10:52
- Changed state: **Reproducing**

**Comment by GoogleCodeExporter** [comment]
```
Do you see a link "Account Administration" in settings section?
```

Original comment by `rol...@liebl.ath.cx` on 18 May 2012 at 10:53

**Comment by GoogleCodeExporter** [comment]
```
Feedback? Closing soon as invalid.
```

Original comment by `rol...@liebl.ath.cx` on 20 May 2012 at 2:37

**Comment by GoogleCodeExporter** [comment]
```
Hi, and sorry for the delay,

Yes I am using hMailserver and it is on the same box as the Roundcube
installation, and no, there is no "Account Administration" link in the
settings section, even though the "change password" plugin is installed and
selected.

Again, sorry for the delay.
```

Original comment by `miket...@gmail.com` on 28 May 2012 at 8:09

**Comment by GoogleCodeExporter** [comment]
```
Sorry for delay on my side. Is this still an issue?
```

Original comment by `rol...@liebl.ath.cx` on 25 Jun 2012 at 2:56

**Comment by GoogleCodeExporter** [comment]
```
Yes, though no actions were taken from my side either. Is there any more
information I can give to help you?
```

Original comment by `miket...@gmail.com` on 26 Jun 2012 at 7:04

**Comment by GoogleCodeExporter** [comment]
```
Did you rename config.inc.php.dist to config.inc.php?

If you want me to check your setup, I need an IMAP Test-Account and 
FTP-/SSH-Access to Roundcube Root (read/write). Mail me details to myroundcube 
at mail4us dot net.


```

Original comment by `rol...@liebl.ath.cx` on 26 Jun 2012 at 8:19

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I have deleted comment #9. You have exposed the access data by sending it to 
the mailing list to the whole world. Please change passwords immediately and 
MAIL ME the details (myroundcube at mail4us dot net).
```

Original comment by `rol...@liebl.ath.cx` on 26 Jun 2012 at 1:37

**Comment by GoogleCodeExporter** [comment]
```
hmail_password was never loaded because you
have binded the plugin in plugin_manager config
to wrong hosts:

  'misc' => array(
    'hmail_password' => array( 
      'label_name' => 'hmail_password.changepasswd',
      'label_description' => 'hmail_password.plugindescription',
      'protected' => false,
      'hosts' => array('mail4us.net', 'svn.mail4us.net'),
      'active' => true
    ),

see 'hosts'.

I'll fix plugin_manager not to show the plugin checkbox
if this part is mis-configured. So, let's say both of us
made an mistake:

#1- You mis-configured it.
#2- plugin_manager should not show the plugin checkbox
    if plugin is not binded to the host.

On you server there is now:

  'misc' => array(
    'hmail_password' => array( 
      'label_name' => 'hmail_password.changepasswd',
      'label_description' => 'hmail_password.plugindescription',
      'protected' => false,
      //'hosts' => array('mail4us.net', 'svn.mail4us.net'),
      'active' => true
    ),

... 'hosts' is commented !!!

--------------

Label change to plugin_manager / subject edited
```

Original comment by `rol...@liebl.ath.cx` on 29 Jun 2012 at 5:30
- Changed title: **[plugin_manager] - hmail_password in plugin_manager enabled but Account Administration link is not showing up**

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 29 Jun 2012 at 5:30
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in plugin_manager 3.7.
```

Original comment by `rol...@liebl.ath.cx` on 30 Jun 2012 at 4:01
- Changed state: **Fixed**
